### PR TITLE
update hardware requirement for disaggregated tiflash

### PR DESCRIPTION
**Pull request by guo-shaoge**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/16642
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: hardware-and-software-requirements.md [hunk 1]
```diff
@@ -176,3 +176,32 @@ TiDB 作为开源一栈式实时 HTAP 数据库，其正常运行需要网络环
...
+| **组件** | **CPU** | **内存** | **本地存储** | **网络** | **实例数量(最低要求)** |
+| --- | --- | --- | --- | --- | --- |
+| TiFlash 存储节点 | 16 核+ | 32 GB+ | SSD, 200 GB+ | 千兆网卡 | 1 |
+| TiFlash 计算节点 | 16 核+ | 32 GB+ | SSD, 100 GB+ | 千兆网卡 | 0 |
+
+### 生产环境
+
+| **组件** | **CPU** | **内存** | **硬盘类型** | **网络** | **实例数量(最低要求)** |
+| --- | --- | --- | --- | --- | --- |
+| TiFlash 存储节点 | 32 核+ | 64 GB+ | 1 or more SSDs | 万兆网卡（2 块最佳） | 1 |
```
Comment (JaySon-Huang) [inline hardware-and-software-requirements.md:202]:
```suggestion
| TiFlash 存储节点 | 32 核+ | 64 GB+ | SSD, 200 GB+ | 万兆网卡（2 块最佳） | 2 |
```

File: hardware-and-software-requirements.md [hunk 2]
```diff
@@ -176,3 +176,32 @@ TiDB 作为开源一栈式实时 HTAP 数据库，其正常运行需要网络环
...
+| TiFlash 计算节点 | 32 核+ | 64 GB+ | 1 or more SSDs | 万兆网卡（2 块最佳） | 0 |
```
Comment (JaySon-Huang) [inline hardware-and-software-requirements.md:203]: 计算节点现在应该不支持多盘作为 cache 路径

File: hardware-and-software-requirements.md [hunk 3]
```diff
@@ -176,3 +176,32 @@ TiDB 作为开源一栈式实时 HTAP 数据库，其正常运行需要网络环
...
 TiDB 提供了基于 [Grafana](https://grafana.com/) 的技术平台，对数据库集群的各项指标进行可视化展现。采用支持 Javascript 的微软 Edge、Apple Safari、Google Chrome、Mozilla Firefox 的较新版本即可访问监控入口。
+
+## 存算分离 TiFlash 软硬件要求
+
+以上所有描述都是针对存算一体 TiFlash， 从 v7.0.0 开始，TiFlash 开始支持[存算分离](/tiflash/tiflash-disaggregated-and-s3.md)架构, 该架构下 TiFlash 分为存储节点和计算节点两个角色。对应的软硬件要求如下:
+
+1. 软件: 操作系统以及平台要求与存算一体架构一致。
+2. 网络端口: 网络端口要求都与存算一体一致。
+3. 磁盘空间要求:
+    1. 存储节点: 与存算一体一致，但是需要额外配置一个 S3 兼容的对象存储。
```
Comment (JaySon-Huang) [inline hardware-and-software-requirements.md:187]:
```suggestion
    1. 存储节点: 推荐 200 GB+，在增添 TiFlash 副本、Region 副本迁移的时候作为数据上传 S3 前的本地缓冲区。另外需要一个 S3 兼容的对象存储。
```

File: hardware-and-software-requirements.md [hunk 4]
```diff
@@ -176,3 +176,32 @@ TiDB 作为开源一栈式实时 HTAP 数据库，其正常运行需要网络环
...
+    2. 计算节点: 推荐 100G。计算节点的存储主要用来缓存从存储节点读取的数据以便加速性能，所以有可能看到计算节点的缓存用满，这是正常现象。
```
Comment (zanmato1984) [inline hardware-and-software-requirements.md:188]: 数字与单位格式应与上下文保持一致。
```suggestion
    2. 计算节点: 推荐 100 GB+。计算节点的存储主要用来缓存从存储节点读取的数据以便加速性能，所以有可能看到计算节点的缓存用满，这是正常现象。
```

File: hardware-and-software-requirements.md (hunk 2 above)
Comment (zanmato1984) [inline hardware-and-software-requirements.md:203]:
```suggestion
| TiFlash 计算节点 | 32 核+ | 64 GB+ | SSD, 100 GB+ | 万兆网卡（2 块最佳） | 0 (参见下文“注意”) |
```

File: hardware-and-software-requirements.md [hunk 5]
```diff
@@ -176,3 +176,32 @@ TiDB 作为开源一栈式实时 HTAP 数据库，其正常运行需要网络环
...
+4. CPU 以及内存等其他要求见下节
+
+### 开发及测试环境
+
+| **组件** | **CPU** | **内存** | **本地存储** | **网络** | **实例数量(最低要求)** |
+| --- | --- | --- | --- | --- | --- |
+| TiFlash 存储节点 | 16 核+ | 32 GB+ | SSD, 200 GB+ | 千兆网卡 | 1 |
+| TiFlash 计算节点 | 16 核+ | 32 GB+ | SSD, 100 GB+ | 千兆网卡 | 0 |
```
Comment (zanmato1984) [inline hardware-and-software-requirements.md:196]:
```suggestion
| TiFlash 计算节点 | 16 核+ | 32 GB+ | SSD, 100 GB+ | 千兆网卡 | 0 (参见下文“注意”)  |
```

**Review by zanmato1984 — CHANGES_REQUESTED** [review]

File: hardware-and-software-requirements.md [hunk 6]
```diff
@@ -176,3 +176,32 @@ TiDB 作为开源一栈式实时 HTAP 数据库，其正常运行需要网络环
...
+| --- | --- | --- | --- | --- | --- |
+| TiFlash 存储节点 | 16 核+ | 32 GB+ | SSD, 200 GB+ | 千兆网卡 | 1 |
+| TiFlash 计算节点 | 16 核+ | 32 GB+ | SSD, 100 GB+ | 千兆网卡 | 0(参见下文“注意”) |
+
+### 生产环境
+
+| **组件** | **CPU** | **内存** | **硬盘类型** | **网络** | **实例数量(最低要求)** |
+| --- | --- | --- | --- | --- | --- |
+| TiFlash 存储节点 | 32 核+ | 64 GB+ | SSD, 200 GB+ | 万兆网卡（2 块最佳） | 2 |
+| TiFlash 计算节点 | 32 核+ | 64 GB+ | SSD, 100GB+  | 万兆网卡（2 块最佳） | 0(参见下文“注意”) |
```
Comment (zanmato1984) [inline hardware-and-software-requirements.md:203]:
```suggestion
| TiFlash 计算节点 | 32 核+ | 64 GB+ | SSD, 100 GB+  | 万兆网卡（2 块最佳） | 0(参见下文“注意”) |
```

**Review by zanmato1984 — APPROVED** [review]
lgtm

**Review by JaySon-Huang — APPROVED** [review]
LGTM

**Comment by guo-shaoge** [comment]
@Oreoxmt ptal

File: hardware-and-software-requirements.md [hunk 7]
```diff
@@ -176,3 +176,32 @@ TiDB 作为开源一栈式实时 HTAP 数据库，其正常运行需要网络环
 ## 客户端 Web 浏览器要求
 
 TiDB 提供了基于 [Grafana](https://grafana.com/) 的技术平台，对数据库集群的各项指标进行可视化展现。采用支持 Javascript 的微软 Edge、Apple Safari、Google Chrome、Mozilla Firefox 的较新版本即可访问监控入口。
+
+## 存算分离 TiFlash 软硬件要求
+
+以上所有描述都是针对存算一体 TiFlash， 从 v7.0.0 开始，TiFlash 开始支持[存算分离](/tiflash/tiflash-disaggregated-and-s3.md)架构, 该架构下 TiFlash 分为存储节点和计算节点两个角色。对应的软硬件要求如下:
```
Comment (Oreoxmt) [inline hardware-and-software-requirements.md:182]:
```suggestion
## TiFlash 存算分离架构的软硬件要求

上面的 TiFlash 软硬件要求是针对存算一体架构的。从 v7.0.0 开始，TiFlash 支持[存算分离架构](/tiflash/tiflash-disaggregated-and-s3.md)，该架构下 TiFlash 分为存储节点和计算节点两个角色，对应的软硬件要求如下：
```

File: hardware-and-software-requirements.md [hunk 8]
```diff
@@ -176,3 +176,32 @@ TiDB 作为开源一栈式实时 HTAP 数据库，其正常运行需要网络环
...
+
+1. 软件: 操作系统以及平台要求与存算一体架构一致。
+2. 网络端口: 网络端口要求与存算一体一致。
+3. 磁盘空间要求:
+    1. 存储节点: 推荐 200 GB+，在增添 TiFlash 副本、Region 副本迁移的时候作为数据上传 S3 前的本地缓冲区。另外需要一个 S3 兼容的对象存储。
+    2. 计算节点: 推荐 100 GB+。计算节点的存储主要用来缓存从存储节点读取的数据以便加速性能，所以有可能看到计算节点的缓存用满，这是正常现象。
+4. CPU 以及内存等其他要求见下节
```
Comment (Oreoxmt) [inline hardware-and-software-requirements.md:189]:
```suggestion
- 软件：与存算一体架构一致，详见[操作系统及平台要求](#操作系统及平台要求)。
- 网络端口：与存算一体架构一致，详见[网络要求](#网络要求)。
- 磁盘空间：
    - 存储节点：推荐 200 GB+，用作增加 TiFlash 副本、Region 副本迁移时向 Amazon S3 上传数据前的本地缓冲区。此外，还需要一个与 Amazon S3 兼容的对象存储。
    - 计算节点：推荐 100 GB+，主要用于缓存从存储节点读取的数据以提升性能。计算节点的缓存可能会被完全使用，这是正常现象。
- CPU 以及内存等要求参考下文。
```

File: hardware-and-software-requirements.md [hunk 9]
```diff
@@ -176,3 +176,32 @@ TiDB 作为开源一栈式实时 HTAP 数据库，其正常运行需要网络环
...
+
+### 开发及测试环境
+
+| **组件** | **CPU** | **内存** | **本地存储** | **网络** | **实例数量(最低要求)** |
+| --- | --- | --- | --- | --- | --- |
+| TiFlash 存储节点 | 16 核+ | 32 GB+ | SSD, 200 GB+ | 千兆网卡 | 1 |
+| TiFlash 计算节点 | 16 核+ | 32 GB+ | SSD, 100 GB+ | 千兆网卡 | 0(参见下文“注意”) |
```
Comment (Oreoxmt) [inline hardware-and-software-requirements.md:196]:
```suggestion
| 组件 | CPU | 内存 | 本地存储 | 网络 | 实例数量（最低要求） |
| --- | --- | --- | --- | --- | --- |
| TiFlash 存储节点 | 16 核+ | 32 GB+ | SSD, 200 GB+ | 千兆网卡 | 1 |
| TiFlash 计算节点 | 16 核+ | 32 GB+ | SSD, 100 GB+ | 千兆网卡 | 0（参见下文“注意”说明） |
```

File: hardware-and-software-requirements.md [hunk 10]
```diff
@@ -176,3 +176,32 @@ TiDB 作为开源一栈式实时 HTAP 数据库，其正常运行需要网络环
...
+
+### 生产环境
+
+| **组件** | **CPU** | **内存** | **硬盘类型** | **网络** | **实例数量(最低要求)** |
+| --- | --- | --- | --- | --- | --- |
+| TiFlash 存储节点 | 32 核+ | 64 GB+ | SSD, 200 GB+ | 万兆网卡（2 块最佳） | 2 |
+| TiFlash 计算节点 | 32 核+ | 64 GB+ | SSD, 100 GB+  | 万兆网卡（2 块最佳） | 0(参见下文“注意”) |
```
Comment (Oreoxmt) [inline hardware-and-software-requirements.md:203]:
```suggestion
| 组件 | CPU | 内存 | 硬盘类型 | 网络 | 实例数量（最低要求） |
| --- | --- | --- | --- | --- | --- |
| TiFlash 存储节点 | 32 核+ | 64 GB+ | SSD, 200 GB+ | 万兆网卡（2 块最佳） | 2 |
| TiFlash 计算节点 | 32 核+ | 64 GB+ | SSD, 100 GB+  | 万兆网卡（2 块最佳） | 0（参见下文“注意”说明） |
```

File: hardware-and-software-requirements.md [hunk 11]
```diff
@@ -176,3 +176,32 @@ TiDB 作为开源一栈式实时 HTAP 数据库，其正常运行需要网络环
...
+
+> **注意：**
+>
+> - TiFlash 计算节点可以使用 TiUP 等部署工具快速扩缩容，扩缩容范围是 [0, inf+]
```
Comment (Oreoxmt) [inline hardware-and-software-requirements.md:207]:
```suggestion
> TiFlash 计算节点可以使用 TiUP 等部署工具快速扩缩容，扩缩容范围是 `[0, +inf]`。
```

File: hardware-and-software-requirements.md [hunk 12]
```diff
@@ -176,3 +176,32 @@ TiDB 作为开源一栈式实时 HTAP 数据库，其正常运行需要网络环
...
+| --- | --- | --- | --- | --- | --- |
+| TiFlash 存储节点 | 16 核+ | 32 GB+ | SSD, 200 GB+ | 千兆网卡 | 1 |
+| TiFlash 计算节点 | 16 核+ | 32 GB+ | SSD, 100 GB+ | 千兆网卡 | 0（参见下文“注意”说明） |
+
+### 生产环境
+
+| 组件 | CPU | 内存 | 硬盘类型 | 网络 | 实例数量（最低要求） |
+| --- | --- | --- | --- | --- | --- |
+| TiFlash 存储节点 | 32 核+ | 64 GB+ | SSD, 200 GB+ | 万兆网卡（2 块最佳） | 2 |
+| TiFlash 计算节点 | 32 核+ | 64 GB+ | SSD, 100 GB+  | 万兆网卡（2 块最佳） | 0（参见下文“注意”说明） |
```
Comment (Oreoxmt) [inline hardware-and-software-requirements.md:203]:
```suggestion
| TiFlash Compute Node | 32 核+ | 64 GB+ | SSD, 100 GB+  | 万兆网卡（2 块最佳） | 0（参见下文“注意”说明） |
```

File: hardware-and-software-requirements.md [hunk 13]
```diff
@@ -176,3 +176,32 @@ TiDB 作为开源一栈式实时 HTAP 数据库，其正常运行需要网络环
...
+
+> **注意：**
+>
+> TiFlash 计算节点可以使用 TiUP 等部署工具快速扩缩容，扩缩容范围是 `[0, +inf]`。
```
Comment (Oreoxmt) [inline hardware-and-software-requirements.md:207]:
```suggestion
> TiFlash Compute Node 可以使用 TiUP 等部署工具快速扩缩容，扩缩容范围是 `[0, +inf]`。
```

File: hardware-and-software-requirements.md [hunk 14]
```diff
@@ -176,3 +176,32 @@ TiDB 作为开源一栈式实时 HTAP 数据库，其正常运行需要网络环
...
+| 组件 | CPU | 内存 | 本地存储 | 网络 | 实例数量（最低要求） |
+| --- | --- | --- | --- | --- | --- |
+| TiFlash 存储节点 | 16 核+ | 32 GB+ | SSD, 200 GB+ | 千兆网卡 | 1 |
+| TiFlash 计算节点 | 16 核+ | 32 GB+ | SSD, 100 GB+ | 千兆网卡 | 0（参见下文“注意”说明） |
+
+### 生产环境
+
+| 组件 | CPU | 内存 | 硬盘类型 | 网络 | 实例数量（最低要求） |
+| --- | --- | --- | --- | --- | --- |
+| TiFlash 存储节点 | 32 核+ | 64 GB+ | SSD, 200 GB+ | 万兆网卡（2 块最佳） | 2 |
```
Comment (Oreoxmt) [inline hardware-and-software-requirements.md:202]:
```suggestion
| TiFlash Write Node | 32 核+ | 64 GB+ | SSD, 200 GB+ | 万兆网卡（2 块最佳） | 2 |
```

File: hardware-and-software-requirements.md [hunk 15]
```diff
@@ -176,3 +176,32 @@ TiDB 作为开源一栈式实时 HTAP 数据库，其正常运行需要网络环
...
+    - 存储节点：推荐 200 GB+，用作增加 TiFlash 副本、Region 副本迁移时向 Amazon S3 上传数据前的本地缓冲区。此外，还需要一个与 Amazon S3 兼容的对象存储。
+    - 计算节点：推荐 100 GB+，主要用于缓存从存储节点读取的数据以提升性能。计算节点的缓存可能会被完全使用，这是正常现象。
+- CPU 以及内存等要求参考下文。
+
+### 开发及测试环境
+
+| 组件 | CPU | 内存 | 本地存储 | 网络 | 实例数量（最低要求） |
+| --- | --- | --- | --- | --- | --- |
+| TiFlash 存储节点 | 16 核+ | 32 GB+ | SSD, 200 GB+ | 千兆网卡 | 1 |
+| TiFlash 计算节点 | 16 核+ | 32 GB+ | SSD, 100 GB+ | 千兆网卡 | 0（参见下文“注意”说明） |
```
Comment (Oreoxmt) [inline hardware-and-software-requirements.md:196]:
```suggestion
| TiFlash Write Node | 16 核+ | 32 GB+ | SSD, 200 GB+ | 千兆网卡 | 1 |
| TiFlash Compute Node | 16 核+ | 32 GB+ | SSD, 100 GB+ | 千兆网卡 | 0（参见下文“注意”说明） |
```

File: hardware-and-software-requirements.md [hunk 16]
```diff
@@ -176,3 +176,32 @@ TiDB 作为开源一栈式实时 HTAP 数据库，其正常运行需要网络环
...
+
+## TiFlash 存算分离架构的软硬件要求
+
+上面的 TiFlash 软硬件要求是针对存算一体架构的。从 v7.0.0 开始，TiFlash 支持[存算分离架构](/tiflash/tiflash-disaggregated-and-s3.md)，该架构下 TiFlash 分为存储节点和计算节点两个角色，对应的软硬件要求如下：
+
+- 软件：与存算一体架构一致，详见[操作系统及平台要求](#操作系统及平台要求)。
+- 网络端口：与存算一体架构一致，详见[网络要求](#网络要求)。
+- 磁盘空间：
+    - 存储节点：推荐 200 GB+，用作增加 TiFlash 副本、Region 副本迁移时向 Amazon S3 上传数据前的本地缓冲区。此外，还需要一个与 Amazon S3 兼容的对象存储。
+    - 计算节点：推荐 100 GB+，主要用于缓存从存储节点读取的数据以提升性能。计算节点的缓存可能会被完全使用，这是正常现象。
```
Comment (Oreoxmt) [inline hardware-and-software-requirements.md:188]:
```suggestion
    - TiFlash Write Node：推荐 200 GB+，用作增加 TiFlash 副本、Region 副本迁移时向 Amazon S3 上传数据前的本地缓冲区。此外，还需要一个与 Amazon S3 兼容的对象存储。
    - TiFlash Compute Node：推荐 100 GB+，主要用于缓存从 Write Node 读取的数据以提升性能。Compute Node 的缓存可能会被完全使用，这是正常现象。
```

File: hardware-and-software-requirements.md [hunk 17]
```diff
@@ -176,3 +176,32 @@ TiDB 作为开源一栈式实时 HTAP 数据库，其正常运行需要网络环
 ## 客户端 Web 浏览器要求
 
 TiDB 提供了基于 [Grafana](https://grafana.com/) 的技术平台，对数据库集群的各项指标进行可视化展现。采用支持 Javascript 的微软 Edge、Apple Safari、Google Chrome、Mozilla Firefox 的较新版本即可访问监控入口。
+
+## TiFlash 存算分离架构的软硬件要求
+
+上面的 TiFlash 软硬件要求是针对存算一体架构的。从 v7.0.0 开始，TiFlash 支持[存算分离架构](/tiflash/tiflash-disaggregated-and-s3.md)，该架构下 TiFlash 分为存储节点和计算节点两个角色，对应的软硬件要求如下：
```
Comment (Oreoxmt) [inline hardware-and-software-requirements.md:182]:
```suggestion
上面的 TiFlash 软硬件要求是针对存算一体架构的。从 v7.0.0 开始，TiFlash 支持[存算分离架构](/tiflash/tiflash-disaggregated-and-s3.md)，该架构下 TiFlash 分为 Write Node 和 Compute Node 两个角色，对应的软硬件要求如下：
```

Comment (Oreoxmt) [reply hardware-and-software-requirements.md:182]: 与 TiFlash 存算分离架构与 S3 支持的文档描述保持一致
https://docs.pingcap.com/zh/tidb/dev/tiflash-disaggregated-and-s3

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**